indoorplants/validation/crossvalidate.py

```diff
@@ -106,8 +106,8 @@ def cv_engine(X, y, model_obj, score_funcs, splits=5, scale_obj=None,
         y_train = y.iloc[train]
         y_test = y.iloc[test]
 
-        X_train = X.iloc[train, :]
-        X_test = X.iloc[test, :]
+        X_train = X.iloc[train,]
```

Comment:
@xcao38 I am assuming we just don't need the column indexing here? or, is there another reason for removing that?

Reply:
Yeah, I remove the colon here because I run into an `IndexingError: Too many indexers` when I passed in the training data with only one feature (e.g. X.score in the FuzzyScoreClassifier). 

This fix suppose to be on the branch fix/46 I don't know how it gets to here!

Reply:
ah, ok - to be sure, have you tried this with a full DF (i.e. more than one feature)?

Reply:
Yeah, I also tried it with LogisticRegression (using a bigger data set, i.e. more than 1 feature) in the dob machine learning analysis.